cmd/root_test.go

```diff
@@ -131,11 +131,6 @@ func TestRoot_CMDParameterized(t *testing.T) {
 		t.Fatalf("default command use should be \"func\".")
 	}
 
-	usageExample, err := replaceNameInTemplate("func", "example")
```

Comment:
It's too bad that we lose these tests - could `runRootHelp` perhaps be made more easily testable by setting the command's output stream?

Reply:
Yes @lance that's do-able and a good idea.  I have now refactored that test to confirm actual command output contains usage text with a properly substituted command name string.